GeneralStateTests/stEIP2930/storageCosts.json

```diff
@@ -0,0 +1,817 @@
+                            "0x0000000000000000000000000000000000000000000000000000000000000010"
+                        ]
+                    }
+                ],
+                [
+                    {
+                        "address" : "0x0000000000000000000000000000000000000104",
+                        "storageKeys" : [
+                            "0x0000000000000000000000000000000000000000000000000000000000000010"
+                        ]
+                    }
+                ],
+                [
+                    {
+                        "address" : "0x0000000000000000000000000000000000000105",
+                        "storageKeys" : [
+                            "0x0000000000000000000000000000000000000000000000000000000000000010"
+                        ]
+                    }
+                ],
+                [
+                    {
+                        "address" : "0xf000000000000000000000000000000000000100",
+                        "storageKeys" : [
+                            "0x0000000000000000000000000000000000000000000000000000000000000000"
+                        ]
+                    }
+                ],
+                [
+                    {
+                        "address" : "0xf000000000000000000000000000000000000101",
+                        "storageKeys" : [
+                            "0x0000000000000000000000000000000000000000000000000000000000000000"
+                        ]
+                    }
+                ],
+                [
+                    {
+                        "address" : "0xf000000000000000000000000000000000000101",
+                        "storageKeys" : [
+                            "0x0000000000000000000000000000000000000000000000000000000000000000"
+                        ]
+                    }
+                ],
+                [
+                    {
+                        "address" : "0xf000000000000000000000000000000000000101",
+                        "storageKeys" : [
+                            "0x0000000000000000000000000000000000000000000000000000000000000000"
+                        ]
+                    }
+                ],
+                [
+                    {
+                        "address" : "0xf000000000000000000000000000000000000101",
+                        "storageKeys" : [
+                            "0x0000000000000000000000000000000000000000000000000000000000000000"
+                        ]
+                    }
+                ],
+                [
+                    {
+                        "address" : "0xf000000000000000000000000000000000000101",
+                        "storageKeys" : [
+                            "0x0000000000000000000000000000000000000000000000000000000000000000"
+                        ]
+                    }
+                ],
+                {},
+                {},
+                {},
+                {},
+                {},
+                {},
```

Comment:
These are wrong, they should all be empty lists, not empty objects: `[]` .

Reply:
This is correct. The corresponding transactions are legacy transactions

Reply:
There are two ways in which a transaction can not have an effective access list:

1. Transaction type 0, the original type, which doesn't have an access list.
2. Transaction type 1, the new type, with nothing in the access list.

We need to check both options.

Reply:
Changes `{}` to `null`